airflow/lineage/__init__.py

```diff
@@ -108,9 +115,9 @@ def wrapper(self, context, *args, **kwargs):
                       for i in inlets]
             self.inlets.extend(inlets)
 
-        if self._inlets['auto']:
+        if self._inlets['auto']:  # pylint: disable=protected-access
```

Comment:
Do we need this warning disabling here? Surely we are allowed to access protected members of our own instance?!